Extending/FileSystemProviders/Azure-Blob-Storage/index-v7.md

```diff
@@ -68,10 +70,44 @@ If you are using IISExpress (as with Visual Studio) you’ll need to add a stati
     </location>
   </configuration>
 ```
+### Forms
+To clarify we are not storing form submissions into Azure storage but the JSON files used to map the layout of the forms such as the templates used and workflows. Forms submissions are stored in the database.
+
+To configure Umbraco forms data its very similar to how media is setup above. You just need to provide another provider as a sibling to the media provider. In the snippet below I have added a provider with the alias of forms along side with the media provider.
+
+```xml
+<?xml version="1.0"?>
+<FileSystemProviders>
+  <!-- Media -->
+  <Provider alias="media" type="Our.Umbraco.FileSystemProviders.Azure.AzureBlobFileSystem, Our.Umbraco.FileSystemProviders.Azure"></Provider>
+  <!-- Forms -->
+	<Provider alias="forms" type="Our.Umbraco.FileSystemProviders.Azure.AzureBlobFileSystem, Our.Umbraco.FileSystemProviders.Azure">
+    <Parameters>
+    <add key="containerName" value="forms"/>
+    <add key="rootUrl" value="http://[myAccountName].blob.core.windows.net/"/>
+    <add key="connectionString" value="DefaultEndpointsProtocol=https;AccountName=[myAccountName];AccountKey=[myAccountKey]"/>
+    <!--
+        Optional configuration value determining the maximum number of days to cache items in the browser.
+        Defaults to 365 days.
+    -->
+    <add key="maxDays" value="365"/>
+    <!--
+        When true this allows the VirtualPathProvider to use the default "media" route prefix regardless
+        of the container name.
+    -->
+    <add key="useDefaultRoute" value="true"/>
+    <!--
+        When true blob containers will be private instead of public what means that you can't access the original blob file directly from its blob url.
+    -->
+    <add key="usePrivateContainer" value="true"/>
```

Comment:
So, setting this to `true` means that the folder can't be accessed? Is that correct?

Reply:
Yes forms data shouldn't be publicly accessible unlike media. By setting this to true we confirm that we are using private blob containers.